torch_frame/data/dataset.py

```diff
@@ -91,13 +117,15 @@ def canonicalize_col_to_pattern(col_to_pattern: Any | dict[str, Any] | None,
         for col in columns:
             col_to_pattern[col] = pattern
     else:
-        if all_inclusive and len(set(columns) -
-                                 set(col_to_pattern.keys())) > 0:
-            raise ValueError(
-                "col_to_sep needs to specify separators for all "
-                "multi-categorical columns, but the separators for the "
-                "following columns are missing: "
-                f"{list(set(columns) - set(col_to_pattern.keys()))}.")
+        missing_cols = set(columns) - set(col_to_pattern.keys())
+        if len(missing_cols) > 0:
+            if requires_all_inclusive:
+                raise ValueError(
+                    f"col_to_xxx requires all columns to be specified but the "
```

Comment:
This `col_to_xxx` seems a little bit weird

Reply:
Good point. will modify.